packages/fxa-content-server/tests/server/routes/get-index.js

```diff
@@ -6,6 +6,8 @@ const assert = intern.getPlugin('chai').assert;
 const sinon = require('sinon');
 const route = require('../../../server/lib/routes/get-index');
 const config = require('../../../server/lib/configuration');
+const surveysFromFile = require('../../../server/config/surveys.json');
```

Comment:
This would still require editing, pushing, and building in order update the list of survey correct?  Is FXA-2002 going to change this?